modules/eks-managed-node-group/main.tf

```diff
@@ -379,9 +382,6 @@ resource "aws_eks_node_group" "this" {
 
   lifecycle {
     create_before_destroy = true
-    ignore_changes = [
-      scaling_config[0].desired_size,
```

Comment:
Could we describe why this change was needed?

Reply:
We encountered an issue before where `desired_size` changes in the code were ignored and had no effect, so it was impossible to change the `desired_size` for a node group from code.

Related upstream issue https://github.com/terraform-aws-modules/terraform-aws-eks/issues/835
Related internal ticket is SRE-14776, you can find more context and discussions there.

Reply:
We should mention this change explicitly, since it's now inconsistent with the FAQ docs: https://github.com/pinterest/terraform-aws-eks/blob/pinterest-release-19.17.2/docs/faq.md#why-are-there-no-changes-when-a-node-groups-desired_size-is-modified

Reply:
Updated the docs in https://github.com/pinterest/terraform-aws-eks/pull/3/commits/616a00977d4307f80f7015bfd26ab700e2c4e30c